### DISCUSSION TITLE
Metrics in one collumn

### DISCUSSION THREAD

**Discussion by DanCreative** [grafana/grafana-csv-datasource#33] [Ideas]
A nice addition to this plugin, would be the ability to define a metric column whose names would be used instead of Number column names. 
This would be similar to how the SQL Server Grafana plugin does it. 
https://grafana.com/docs/grafana/latest/datasources/mssql/

**Comment by marcusolsson** [comment]
I'm not sure I understand what you'd like to see. Could you give me an example?

**Reply by DanCreative** [reply]
This is what the plugin is does:
![image](https://user-images.githubusercontent.com/54568785/107881854-d3d2a000-6eee-11eb-8bb9-25ba2d2e7591.png)

This is my idea:
![image](https://user-images.githubusercontent.com/54568785/107881875-eb118d80-6eee-11eb-94e4-41bca7af2db1.png)


I don't know if that clears it up.

**Reply by marcusolsson** [reply]
Gotcha! Put in a different way you'd basically would like to group by the values in another field, i.e. splitting a response into several. I agree it would be a nice addition. The main challenge would be to figure a good UX for this. Could you maybe create an issue for this?

**Reply by marcusolsson** [reply]
FWIW I'd love to see this as a generic Transformation in Grafana rather than each data source having to implement it separately.
